sefaria/model/text.py

```diff
@@ -5675,22 +5681,45 @@ def get_simple_term_mapping_json(self, rebuild=False):
         return self._simple_term_mapping_json
 
     def get_term(self, term_name):
+        """
+        Returns the full term, if mapping not present, builds the full term mapping.
+        :param term_name: String
+        :returns: full Term (Mongo Record)
+        """
         if not self._full_term_mapping:
             self.build_term_mappings()
         return self._full_term_mapping.get(term_name) if term_name in self._full_term_mapping else Term().load({"name": term_name})
 
 
 
     def get_topic(self, slug):
+        """
+        Returns a dictionary containing with keys "en" and "he".
```

Comment:
the phrase "containing with keys" doesn't sound grammatically correct to me

Reply:
Thanks for catching that! My bad, a typo.